vectordb_bench/backend/clients/tencent_elasticsearch/tencent_elasticsearch.py

```diff
@@ -0,0 +1,250 @@
+import logging
+import time
+from collections.abc import Iterable
+from contextlib import contextmanager
+
+from elasticsearch.helpers import bulk
+
+from vectordb_bench.backend.filter import Filter, FilterOp
+
+from ..api import VectorDB
+from .config import TencentElasticsearchIndexConfig
+
+for logger in ("elasticsearch", "elastic_transport"):
+    logging.getLogger(logger).setLevel(logging.WARNING)
+
+log = logging.getLogger(__name__)
+
+
+SECONDS_WAITING_FOR_FORCE_MERGE_API_CALL_SEC = 30
+
+
+class TencentElasticsearch(VectorDB):
```

Comment:
I've noticed that the code here is very similar to the one for `ElasticCloud`. You can try to directly inherit from the `ElasticCloud` class and override any operations that are different. For specific practices, you can refer to
https://github.com/zilliztech/VectorDBBench/blob/cdd2a1ee82953e33aa851b8bce24e591333d7abf/vectordb_bench/backend/clients/aliyun_elasticsearch/aliyun_elasticsearch.py#L5-L26